heat_infoblox/resources/grid_member.py

```diff
@@ -50,10 +50,13 @@ class GridMember(resource.Resource):
         DNS_DTC_HEALTH_SOURCE, DNS_DTC_HEALTH_SOURCE_ADDRESS,
         DNS_RPZ_QNAME_WAIT_RECURSE, DNS_USE_RPZ_QNAME_WAIT_RECURSE,
         DNS_LOG_DTC_GSLB, DNS_LOG_DTC_HEALTH, DNS_UNBOUND_LOGGING_LEVEL,
+        HA_PAIR, VIP, VIRTUAL_ROUTER_ID, LAN2_VIRTUAL_ROUTER_ID,
+        NODE1_HA_IP, NODE2_HA_IP, NODE2_LAN1_IP,
```

Comment:
I did it in template - pull request at https://github.com/infobloxopen/engcloud/pull/2
Should I instead add python code to do that?